ptbr/config_cli.py

```
@@ -0,0 +1,741 @@
+    Returns a cleaned dict with validated/defaulted values.
+    """
+    result: Dict[str, Any] = {}
+
+    for key, expected_type, required, default, constraint in rules:
+        field_path = f"{section_prefix}.{key}"
+        raw_value = section_data.get(key)
+
+        # --- Missing / None handling ---
+        if raw_value is None:
+            if required:
+                report.add_error(field_path, f"REQUIRED field is missing or null.")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)